webcompat/static/js/lib/bugform.js

```diff
@@ -284,7 +299,7 @@ function BugForm() {
   };
 
   this.addPreviewBackground = function(dataURI) {
-    if (!_.startsWith(dataURI, 'data:image/png;base64')) {
+    if (!_.startsWith(dataURI, 'data:image/')) {
```

Comment:
This was a goofy bug that meant JPGs and GIFs, etc. wouldn't show image previews. 💩